doc/source/getting_started/introduction.ipynb

```diff
@@ -28,15 +30,15 @@
     "</div>\n",
     "\n",
     "\n",
-    "Let's see how this works in practice.\n",
+    "Let's see how some of these concepts work in practice.\n",
     "\n",
     "## Reading and writing files\n",
     "\n",
     "First, we need to read some data.\n",
     "\n",
-    "### Read files\n",
+    "### Reading files\n",
     "\n",
-    "Assuming we have a file containing both data and geometry (e.g. GeoPackage, GeoJSON, Shapefile), we can easily read it using `geopandas.read_file` function, which automatically detects filetype and creates a `GeoDataFrame`. In this example, we'll use the `\"nybb\"` dataset, a map of New York boroughs which is part of GeoPandas installation. Therefore we need to get the path to the actual file. With your file, you specify a path as a string (`\"my_data/my_file.geojson\"`)."
+    "Assuming you have a file containing both data and geometry (e.g. GeoPackage, GeoJSON, Shapefile), you can read it using `geopandas.read_file()`, passing in the path to the file as string (e.g. `\"my_data/my_file.geojson\"`), and the function automatically detects the filetype and creates a `GeoDataFrame`. This tutorial uses the `\"nybb\"` dataset, a map of New York boroughs, which is part of the GeoPandas installation. Therefore, we use `geopandas.datasets.get_path()` to retrieve the path to the dataset."
```

Comment:
Can we leave out the mention of string here? `read_file()` is more versatile than that and accepts also file-like objects of path objects. Just to avoid potential confusion here.

Reply:
Of course!